optaplanner-quickstarts/quarkus-school-timetabling/build.gradle

```diff
@@ -14,9 +14,10 @@ repositories {
 }
 
 dependencies {
-    implementation enforcedPlatform("io.quarkus:quarkus-universe-bom:1.4.2.Final")
-    implementation 'io.quarkus:quarkus-optaplanner'
-    implementation 'io.quarkus:quarkus-optaplanner-jackson'
+    implementation platform("io.quarkus:quarkus-universe-bom:1.7.0.Final")
+    implementation "org.optaplanner:optaplanner-quarkus:${optaplannerVersion}"
+    implementation "org.optaplanner:optaplanner-quarkus-jackson:${optaplannerVersion}"
+    implementation "org.optaplanner:optaplanner-test:${optaplannerVersion}"
```

Comment:
The gradle build was broken due to this missing test dependency.

Yes, we need CI to somehow run the gradle build for the quickstarts too, maybe from a maven profile in the directory above? Separate issue/PR in any case.

Reply:
@rsynek This separate issue/PR, is this something you could take a look at some time? (no rush)

Reply:
Quarkus does it this way for their gradle support (the quarkus gradle plugin):
https://github.com/quarkusio/quarkus/blob/master/devtools/gradle/pom.xml#L75

For quickstarts, they use only maven.